rdrf/rdrf/testing/unit/tests.py

```diff
@@ -30,8 +30,66 @@
+
+    def test_migrate_cdes_clinicaldata(self):
+
+        #print("Input data:")
+        # print(self.input_data)
+        out_data = tranform_data_dict( ["CDE00016", "FHCRP"], "SEC0005", "SEC0003", self.input_data)
+        #print("Output data:")
+        # print(out_data)
+        #print("Source Section:")
+        s_section = get_section("SEC0005", out_data)
+        # print(s_section)
+        #print("Target Section:")
+        t_section = get_section("SEC0003", out_data)
+        # print(t_section)
+
+        def check_cde_in_section(cde_code, section_dict):
+
+            for item in section_dict['cdes']:
+                # Check if section is multiple or not
+                if isinstance(item, list):
+                    cdes_list = item
+                    for cde_dict_item in cdes_list:
+                        if cde_dict_item['code'] == cde_code:
+                            return True
+                else:
+                    cde_dict_item = item
+                    if cde_dict_item['code'] == cde_code:
+                        return True
+
+        assert not check_cde_in_section("CDE00016", s_section), "CDE00016 is still in source section"
+        assert check_cde_in_section("CDE00016", t_section), "CDE00016 is not in target section found"
+
+        assert not check_cde_in_section("FHCRP", s_section), "FHCRP is still in source section"
+        assert check_cde_in_section("FHCRP", t_section), "FHCRP is not in target section found"
+
+        assert check_cde_in_section("FHCompliance", s_section), "FHCompliance has been moved."
+
+        #assert False is not out_data,"Output data "
+
+    """def tearDown(self):
+        print("Tear down method to delete variables")
+        self.input_data.delete()
```

Comment:
Is this meant to be commented out ?


Just remove this as dictionaries don't have a delete method